### DISCUSSION TITLE
fastapi-users-db-sqlalchemy depends on databases?

### DISCUSSION THREAD

**Discussion by davidbrochart** [fastapi-users/fastapi-users#951] [Q&A] [answered]
While trying to package [fastapi-users-db-sqlalchemy](https://github.com/fastapi-users/fastapi-users-db-sqlalchemy) for [conda-forge](https://github.com/conda-forge/staged-recipes/pull/18425), it looks like [databases](https://github.com/encode/databases) should be added as a dependency.

**Comment by frankie567** [comment, marked as answer]
Hi @davidbrochart!

Actually, it depends! If you want to package a version `<=v1.1.0`, then yes, it's needed: https://github.com/fastapi-users/fastapi-users-db-sqlalchemy/blob/v1.1.0/pyproject.toml#L24-L27

On newer versions, `>=v2.0.0`, we use a pure SQLAlchemy approach so it's not needed.

**Comment by davidbrochart** [comment]
Good to know, I'll remove `databases` in the requirements for [v3.0.0](https://github.com/conda-forge/fastapi-users-db-sqlalchemy-feedstock/pull/1#issuecomment-1075988027).
Thanks!
